.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 💡 Feature Request
+about: I have a suggestion (and might want to implement myself 🙂)!
+---
+
+<!--
+
+Vote on feature requests by adding a 👍. This helps maintainers prioritize what
+to work on.
+
+* Please fill out this template with all the relevant information so we can
+  understand what's going on and fix the issue. We appreciate bugs filed and PRs
+  submitted!
+
+* If you're issue is regarding one of the query APIs (`getByText`
+  `getByLabelText`, etc), then please file it on the `dom-testing-library`
+  repository instead. If you file it here it will be closed. Thanks :)
+
+* Please make sure that you are familiar with and follow the Code of Conduct for
+  this project (found in the CODE_OF_CONDUCT.md file).
+
+We'll probably ask you to submit the fix (after giving some direction). If
+you've never done that before, that's great! Check this free short video
+tutorial to learn how: http://kcd.im/pull-request
+
+-->
+
+### Describe the feature you'd like:
```

Comment:
Could we add a section after this one for "### Suggested implementation" with a comment that says it's helpful but optional?